helm/management-cluster-admission/templates/kustomize-out.yaml

```diff
@@ -498,6 +498,10 @@ webhooks:
       operator: Exists
     - key: app.kubernetes.io/version
       operator: Exists
+    - key: app.kubernetes.io/name
```

Comment:
would it not make sense to export that in the values.yaml file if we have additional exceptions? I think the Azure ad pod identity app will be the same for example

Reply:
This does work now though thanks :D

Reply:
We can do that later. For now I'd like to see how it goes and if we can't go with something generic. Maybe like `vertical-pod-autoscaler-app-ignore/include` label.